### PR TITLE
Chrome and IE LowerParams fix

### LINKED ISSUE #135: Exception occurred when checking element file, check the 'laravelValidation' method
Hello sorry but I have another issue (still researching the other one)

With file uploads I get the following error:

```Exception occurred when checking element file, check the 'laravelValidation' method.```

```javascript
$("#new-info-form").validate({
    errorElement: 'span',
    errorClass: 'help-block error-help-block',
    errorPlacement: function(error, element) {
        if (element.parent('.input-group').length ||
            element.prop('type') === 'checkbox' || element.prop('type') === 'radio') {
            error.insertAfter(element.parent());
        } else {
            error.insertAfter(element);
        }
    },
    highlight: function(element) {
        $(element).closest('.form-group').addClass('has-error');
    },
    success: function(element) {
        $(element).closest('.form-group').removeClass('has-error');
    },
    focusInvalid: false,
    rules: {
        "client_id": {
            "laravelValidation": [
                ["Required", [], "The client id field is required.", true]
            ],
            "laravelValidationRemote": [
                ["Exists", ["client_id", "eyJpdiI6IlhJQTJ4NVd3UlwvT0s0azg5TW5nNDhRPT0iLCJ2YWx1ZSI6IlNzOTZ4MGdrbHpFQzdyN2tMQmR3RG9EeHpUQXYwYUltWFZBOUo3YWpNK1kxdjRMNlJoRkN0UnlQTDEzSERyMENQQ0Q1VlVRM0EyN0gycm51QXNFNzJ3PT0iLCJtYW2VkYTk2NjFjIn0="], "The selected client id is invalid.", false]
            ]
        },
        "file": {
            "laravelValidation": [
                ["Required", [], "The file field is required.", true],
                ["Mimes", ["pdf", "doc", "docx", "doc", "csv", "jpeg", "jpg", "gif", "png", "txt"], "The file must be a file of type: pdf, doc, docx, doc, csv, jpeg, jpg, gif, png, txt.", false]
            ]
        },
        "filename": {
            "laravelValidation": [
                ["Required", [], "The filename field is required.", true],
                ["Max", ["255"], "The filename may not be greater than 255 characters.", false]
            ]
        }
    }
});
```

I have this error on any page with a file upload, and it is either e or b:
```
Uncaught TypeError: e is not a function.  Exception occurred when checking element file, check the 'laravelValidation' method.
```

```
Uncaught TypeError: b is not a function. Exception occurred when checking element image, check the 'laravelValidation' method.
```

### PR DESCRIPTION
**Pull request by adamtester**
Fixes #135

### CODE REVIEW

**Comment by gbaudoin** [comment]
What is the StyleCI rule it is breaking ?

**Comment by adamtester** [comment]
Not in the files I changed, I can fix them if you'd like?

`src/Exceptions/BadRequestHttpException.php`
and
`src/JsValidationServiceProvider.php`

**Comment by torrentalle** [comment]
Yes ples, can you fix it?

Thank's for your support

**Comment by adamtester** [comment]
Should be it once Scrutinizer and Travis pass

File: resources/assets/js/validations.js [hunk 1]
```diff
@@ -376,7 +376,10 @@ $.extend(true, laravelValidation, {
          * @return {boolean}
          */
         Mimes: function(value, element, params) {
-            var lowerParams = $.map(params, String.toLowerCase);
+            var lowerParams = $.map(params, function(item, index) {
```
Comment (torrentalle) [inline resources/assets/js/validations.js]: Codacy Issue found: ["index" is defined but never used](https://www.codacy.com/app/albert-moreno-t/laravel-jsvalidation/file/2183766929/issues/source?bid=2225045&fileBranchId=3117047#l379)

**Pull request merged**